docs/variants/zir_blazer/hardware_matrix.md

```diff
@@ -0,0 +1,52 @@
+# Hardware configuration matrix
+
+The tables below contain information about current components of
+workstations for Zir_Blazer available in 3mdeb testing laboratory.
+
+## LA1E312159
+
+| Component                      | Description                                      |
+|--------------------------------|--------------------------------------------------|
+| **CPU**                        | Intel Core i5-12600K 3.7                         |
+|                                | Noctua NH-U12S CPU Cooler                        |
+| **GPU**                        | N/A                                              |
+| **RAM**                        | Slot 1: Kingston KF436C17BBK4/32                 |
+|                                | Slot 2: Kingston KF436C17BBK4/32                 |
+|                                | Slot 3: Kingston KF436C17BBK4/32                 |
+|                                | Slot 4: Kingston KF436C17BBK4/32                 |
+| **SSD**                        | Intel 670p 512 GB M26472-201 NVME                |
+| **Flash memory**               | TBD                                              |
+| **Network**                    | Local network wired connection                   |
+| **Attached devices**           | 1. SanDisk Ultra  Flair USB 3.0 16 GB            |
+|                                | 2. Kingston DataTraveler 3.1Gen1 16 GB           |
+|                                | 3. SanDisk Ultra microSD HC I 16 GB              |
+|                                | 4. Logitech, Inc. Keyboard K120                  |
+|                                | 5. Dell Mouse MS116p                             |
+|                                | 6. 1-port USB Type-C                             |
+| **Power Supply**               | SeaSonic FOCUS Plus Platinum                     |
+| **Power Control**              | sonOFF                                           |
+| **Remote Testing Environment** | TBD                                              |
+
+## LA1E012608
+
+| Component                      | Description                                      |
+|--------------------------------|--------------------------------------------------|
+| **CPU**                        | Intel Core i5-12600K 3.7                         |
+|                                | Noctua NH-U12S CPU Cooler                        |
+| **GPU**                        | N/A                                              |
+| **RAM**                        | Slot 1: Kingston KF436C17BBK4/32                 |
+|                                | Slot 2: Kingston KF436C17BBK4/32                 |
+|                                | Slot 3: Kingston KF436C17BBK4/32                 |
+|                                | Slot 4: Kingston KF436C17BBK4/32                 |
+| **SSD**                        | Intel 670p 512 GB M26472-201 NVME                |
+| **Flash memory**               | TBD                                              |
```

Comment:
Information about flash memory should be added

Reply:
added